### PR TITLE
feat: added-v3-oa-verify

### PR DESCRIPTION
**Pull request by SiaChenRen**
https://www.pivotaltracker.com/story/show/175695584

[W3C VC] - Release oa-cli for oav3

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "prettier.endOfLine": "auto",
+    "files.eol": "\n"
```
Comment (yehjxraymond) [inline .vscode/settings.json:3]: possible to add this file into .gitignore? Think we typically dont commit this this. It's quite specific to your machine in this case

File: package.json [hunk 2]
```diff
@@ -68,13 +68,13 @@
...
-    "@govtechsg/oa-verify": "^5.1.0",
-    "@govtechsg/open-attestation": "^4.5.0",
-    "@govtechsg/token-registry": "^2.3.0",
+    "@govtechsg/dnsprove": "2.1.1",
+    "@govtechsg/document-store": "2.2.0",
+    "@govtechsg/oa-did-sign": "2.1.0",
+    "@govtechsg/oa-encryption": "1.3.2",
+    "@govtechsg/oa-verify": "7.1.0",
+    "@govtechsg/open-attestation": "4.5.0",
+    "@govtechsg/token-registry": "2.4.0",
```
Comment (yehjxraymond) [inline package.json:77]: How come all the caret went missing? Dont have to fix the version, can use the `^` to allow minor/patch upgrades

File: src/__tests__/verify.v3.e2e.test.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import { handler } from "../commands/verify";
```
Comment (yehjxraymond) [inline src/__tests__/verify.v3.e2e.test.ts:1]: Weird to have this file name if the other one is not labelled v2.

File: src/__tests__/wrap.v3.e2e.test.ts [hunk 4]
```diff
@@ -468,7 +468,7 @@ describe("wrap", () => {
           unwrap: false,
           batched: true,
         });
-        // console.log(fs.readFileSync(outputFile.name, { encoding: "utf8" }));
+        console.log(fs.readFileSync(outputFile.name, { encoding: "utf8" }));
```
Comment (yehjxraymond) [inline src/__tests__/wrap.v3.e2e.test.ts:471]: Is it printing in test? this is quite irritating.. Can remove this altogether?

Comment (SiaChenRen) [reply src/__tests__/wrap.v3.e2e.test.ts:471]: Done.

File: src/__tests__/verify.v3.e2e.test.ts (hunk 3 above)
Comment (SiaChenRen) [reply src/__tests__/verify.v3.e2e.test.ts:1]: Changed verify.e2e.test.ts to verify.v2.e2e.test.ts

File: package.json (hunk 2 above)
Comment (SiaChenRen) [reply package.json:77]: Done

File: .vscode/settings.json (hunk 1 above)
Comment (SiaChenRen) [reply .vscode/settings.json:3]: Done

**Review by yehjxraymond — APPROVED** [review]
LGTM

**Pull request merged**